plugins/github/current_github.mli

```diff
@@ -34,6 +34,15 @@ module Api : sig
     val v : ?description:string -> ?url:Uri.t -> state -> t
   end
 
+  module CheckRunStatus : sig
+    type t
+    type conclusion = [`Failure of string | `Success]
+    type state = [`Queued | `InProgress | `Completed of conclusion]
+
+    (* Construct a CheckRunStatus.t *)
```

Comment:
Doc-comments need `**` (and should go below the `val` line). `dune build @doc` should test it.